sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/RntbdTransportClient.java

```diff
@@ -252,6 +252,8 @@ private void throwIfClosed() {
 
     public static final class Options {
 
+        private static final int DEFAULT_MIN_MAX_CONCURRENT_REQUESTS_PER_ENDPOINT = 10_000;
```

Comment:
should it be DEFAULT_MAX_*?

Reply:
No - it is the minimum allowed value for MaxConcurrentRequestsPerEndpoint (#maxChannel * #maxRequestsPerChannel is used if it is larger than the Default_Min_Max*